File: python/warunek.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

def main(args):
    
    a = int(input("Podaj Pierwszą Liczbę: "))
    b = int(input("Podaj Drugą Liczbę: "))
    c = int(input("Podaj Trzecią Liczbę: "))
    x = 0
    
    print(a, b, c)
    
    if a > b and a > c:
        x = a
    elif b > a and b > c:
        x = b
    elif c > a and c > b:
        x = c
    
    print("Największa Liczba to: ", x)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/warunek.cpp
/*
 * warunek.cpp
 */

#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	int a, b, c;
    int x = 0;
    cout << "Podaj pierwszą liczbę: ";
    cin >> a;
    cout << "Podaj drugą liczbę: ";
    cin >> b;
    cout << "Podaj trzecią liczbę: ";
    cin >> c;
    
    cout << a << " " << b << " " << c << endl;
    if(a > b && a > c){
        x = a;
    }
    else if(b > a && b > c){
        x = b;
    }
    else if(c > a && c > b){
        x = c;
    }
    cout << "Największa liczba: " << x;
    
    
	return 0;
}

